libs/api/domains/documents/src/lib/documentV2.resolver.ts

```diff
@@ -65,34 +78,74 @@ export class DocumentResolverV2 {
     @Args('input') input: DocumentsInput,
     @CurrentUser() user: User,
   ): Promise<PaginatedDocuments> {
-    return this.documentServiceV2.listDocuments(user.nationalId, input)
+    try {
+      return this.documentServiceV2.listDocuments(user.nationalId, input)
+    } catch (e) {
+      this.logger.error('failed to get document list', {
+        category: LOG_CATEGORY,
+        error: e,
+      })
+      throw e
+    }
   }
 
   @ResolveField('categories', () => [Category])
   documentCategories(@CurrentUser() user: User): Promise<Array<Category>> {
-    return this.documentServiceV2.getCategories(user.nationalId)
+    try {
+      return this.documentServiceV2.getCategories(user.nationalId)
+    } catch (e) {
+      this.logger.error('failed to get document categories', {
+        category: LOG_CATEGORY,
+        error: e,
+      })
+      throw e
```

Comment:
all these might be unnecessary, the enhanced fetch should log all this stuff. This isn't adding anything of value to the stack